designsafe/static/scripts/rapid/html/index.html

```diff
@@ -4,119 +4,68 @@
-        <label> End date </label>
-        <input type="date"
-              class="form-control"
-              ng-model="vm.opentopo_filter_options.ot_end_date"
-              ng-change="vm.search()"
-              ng-disabled="!vm.ot_data_source"
-              title="Select the data source to enable filters">
-      </div>
-      <div class="data_source">
-    <input type="checkbox"
-          ng-change="vm.search()"
-          ng-model="vm.ot_data_source">
-    <label class="external_datasource_listing"> External Data Source 2 </label>
-  </div>
-  <div class="filter-options">
-      <label> Keyword search<span class="info-icon" data-tooltip="Keywords typically include: title terms, location details, type of events, index terms associated with the datasets"> (i)</span></label> </label>
-      <input type="text" 
+            ng-model="vm.ot_data_source"/>
+          <label> OpenTopography Data </label>
+        </div>
+        <div class="filter-options">
+          <label> Keyword search<span class="info-icon" data-tooltip="Keywords typically include: title terms, location details, type of events, index terms associated with the datasets"> (i)</span></label> </label>
+          <input type="text" 
             class="form-control" 
             ng-model="vm.opentopo_filter_options.keyword" 
-            ng-change="vm.debouncedSearch()"
             ng-disabled="!vm.ot_data_source"
             title="Select the data source to enable filters"></input>
-      <label> Start date </label>
-      <input type="date"
+          <label> Start date </label>
+          <input type="date"
             class="form-control"
             ng-model="vm.opentopo_filter_options.ot_start_date"
-            ng-change="vm.search()"
             ng-disabled="!vm.ot_data_source"
             title="Select the data source to enable filters">
-      <label> End date </label>
-      <input type="date"
+          <label> End date </label>
+          <input type="date"
             class="form-control"
             ng-model="vm.opentopo_filter_options.ot_end_date"
-            ng-change="vm.search()"
             ng-disabled="!vm.ot_data_source"
             title="Select the data source to enable filters">
-  </div>
-  </div>
-      <div class="filter-options">
-    <button class="btn btn-xs btn-info clear-btn" ng-click="vm.clear_filters()"> Clear </button>
-    <button class="btn btn-xs btn-default hide-btn" ng-click="vm.show_filter_options=false"> Hide </button>
-  </div>
-  </div>
-</ul>
+        </div>
+        </ul>
+      <div class="filter-button-row">
+        <button class="btn btn-xs btn-default filter-btn" ng-click="vm.show_filter_options=false"> Hide Filters </button>
+        <button class="btn btn-xs btn-default filter-btn" ng-click="vm.clear_filters()"> Clear Filters </button>
+        <button class="btn btn-xs btn-info filter-btn" ng-click="vm.debouncedSearch()" ng-disabled="(!vm.data_source && !vm.ot_data_source)"> Search </button>
```

Comment:
Since there is a proper search button now we don't have to worry about debouncing the search so we should be able to use search() directly

Reply:
The search in the sidebar still uses this function. Should that change to the search method instead as well? I think the auto-updating in that area makes sense but we will probably have to defer to CMD on that UI anyway. I can remove it and implement the search the same way as the filter though if you think that would be better.